modules/custom/core/govcms_media/config/install/field.field.media.file.field_media_file.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_file
+    - media.type.file
+  module:
+    - file
+id: media.file.field_media_file
+field_name: field_media_file
+entity_type: media
+bundle: file
+label: File
+description: ''
+required: true
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  file_extensions: 'txt doc docx pdf'
```

Comment:
Do we need to ensure that the list of filetypes in
entity_browser.browser.govcms_media_file_browser.yml and 
entity_browser.browser.govcms_media_image_browser.yml
are matched in
field.field.media.file.field_media_file.yml and
field.field.media.image.field_media_image.yml

It should probably include csv xlsx and pptx at the minimum - but reference our current list at https://github.com/govCMS/govCMS/blob/43606d89a597c7cb38341a956ed799d02e00890a/modules/custom/govcms_core/govcms_core.install#L27

Reply:
@tobybellwood 
based from this list: https://github.com/govCMS/govCMS/blob/43606d89a597c7cb38341a956ed799d02e00890a/modules/custom/govcms_core/govcms_core.install#L27

Below is a breakdown of the extensions and their media type.

image: jpg jpeg gif png svg webp
file: txt doc docx xls xlsx xlsm pdf ppt pptx pps ppsx odt ods odp 
audio: mp3 ogg weba oga
video: mov mp4 m4a m4v mpeg avi ogv webm